src/mappings/multiTokens/events/token_account_destroyed.ts

```diff
@@ -47,36 +48,37 @@ export async function tokenAccountDestroyed(
     skipSave: boolean
 ): Promise<EventModel | undefined> {
     const data = getEventData(ctx, item.event)
-    if (!data) return undefined
 
     if (skipSave) return getEvent(item, data)
 
-    const collectionAccount = await ctx.store.findOne<CollectionAccount>(CollectionAccount, {
-        where: { id: `${data.collectionId}-${u8aToHex(data.accountId)}` },
+    const collectionAccount = await ctx.store.findOneBy<CollectionAccount>(CollectionAccount, {
+        id: `${data.collectionId}-${u8aToHex(data.accountId)}`,
     })
 
-    if (collectionAccount) {
-        collectionAccount.accountCount -= 1
-        await ctx.store.save(collectionAccount)
-    } else {
+    if (collectionAccount === null) {
```

Comment:
why are we checking this way?

Reply:
I inverted the logic and removed the else.

Reply:
Why it is a null instead of !collectionAccount, i'm changing a few stuff and noticed that it is faster to read and do not miss the ! as there are some classes that are very similar but the logic is the opposite.